website/content/docs/v0.15/Introduction/getting-started.md

```diff
@@ -371,8 +350,8 @@ We would set those in the `talosconfig` with:
 
 The node is the target node on which you wish to perform the API call.
 
-Keep in mind, when specifying nodes that their IPs and/or hostnames are as seen by the endpoint servers, not as from the client.
-This is because all connections are proxied first through the endpoints.
+Keep in mind, when specifying nodes that their IPs and/or hostnames are *as seen by the endpoint servers*, not as from the client.
```

Comment:
"Keep in mind, when specifying nodes_,_ that their ..." (missing comma at the end of the clause).